packages/common/test/search/_internal/portalSearchItems.test.ts

```diff
@@ -73,6 +73,44 @@ describe("portalSearchItems Module:", () => {
       );
       expect(hubSite?.properties).not.toBeDefined();
 
+      expect(searchItemsSpy.calls.count()).toBe(1, "should call searchItems");
+      const [expectedParams] = searchItemsSpy.calls.argsFor(0);
+      expect(expectedParams.portal).toEqual(opts.requestOptions?.portal);
+      // 'Code Attachment' comes from default item predicate
+      expect(expectedParams.q).toEqual('(water) AND (-type:"Code Attachment")');
+      expect(expectedParams.countFields).not.toBeDefined();
+    });
+    it("search query does not contain default item predicate for non-item entities", async () => {
+      const searchItemsSpy = spyOn(Portal, "searchItems").and.callFake(() => {
+        return Promise.resolve(cloneObject(AllTypesResponse));
+      });
+
+      const qry: IQuery = {
+        targetEntity: "group",
```

Comment:
See my comment above. I know you have to do this to get 100% code coverage, but by the time we are within `portalSearchItems`, we are guaranteed to have `targetEntity: "item"`. Go ahead and remove this test.